openfisca_us/tools/backdate_parameters.py

```diff
@@ -0,0 +1,36 @@
+from openfisca_core.model_api import *
+from openfisca_core import periods
+
+
+def str_to_instant(s):
+    return periods.Instant(tuple(map(lambda s: int(s), s.split("-"))))
+
+
+def backdate_parameters(
```

Comment:
Is this connected to the CDCC part?

If not, could you describe the problem this solves in an issue, and connect it to this PR?